update Jellyfin to 10.8.11

ParthJadhav opened the pull request:
Small update by Jellyfin providing bug fixes and minor improvements.

nmfretz commented:
Wonderful. Tested on arm64 and x86(amd64). Going live 🪼.

FYI I made a couple small tweaks to the release notes:
1) updated the wording of the security advisory from our last update to make it clear that users on version 10.8.9 and below are affected. Good call on leaving it in for this update, since some users may not have updated last time.
2) I added some white space in the release notes sections so that paragraphs are separated nicely.

With the way that yaml is parsed to render text in the UI, the yaml needs **two** blank newlines between text blocks in order for those text blocks to have a single blank line between them.

Before ([0a5e167](https://github.com/getumbrel/umbrel-apps/pull/796/commits/0a5e1675f2acb74ac8cd5d530b4b9e0d1f952078)):
<img width="1282" alt="image" src="https://github.com/getumbrel/umbrel-apps/assets/85373263/41900bb8-34d0-4921-9056-a1080cc0f586">

After ([9325d70](https://github.com/getumbrel/umbrel-apps/pull/796/commits/9325d70847ee36de1400622b20850e627702aeb8)):
<img width="1282" alt="image" src="https://github.com/getumbrel/umbrel-apps/assets/85373263/4a324cb9-0433-4cea-92b0-e73d523abcef">

ParthJadhav commented:
> Wonderful. Tested on arm64 and x86(amd64). Going live 🪼.
> 
> FYI I made a couple small tweaks to the release notes:
> 
> 1. updated the wording of the security advisory from our last update to make it clear that users on version 10.8.9 and below are affected. Good call on leaving it in for this update, since some users may not have updated last time.
> 2. I added some white space in the release notes sections so that paragraphs are separated nicely.
> 
> With the way that yaml is parsed to render text in the UI, the yaml needs **two** blank newlines between text blocks in order for those text blocks to have a single blank line between them.
> 
> Before ([0a5e167](https://github.com/getumbrel/umbrel-apps/pull/796/commits/0a5e1675f2acb74ac8cd5d530b4b9e0d1f952078)): <img alt="image" width="1282" src="https://user-images.githubusercontent.com/85373263/273066894-41900bb8-34d0-4921-9056-a1080cc0f586.png">
> 
> After ([9325d70](https://github.com/getumbrel/umbrel-apps/pull/796/commits/9325d70847ee36de1400622b20850e627702aeb8)): <img alt="image" width="1282" src="https://user-images.githubusercontent.com/85373263/273069135-4a324cb9-0433-4cea-92b0-e73d523abcef.png">

Gotcha, Thanks !
